cl/corpus_importer/management/commands/recap_into_opinions.py

```diff
@@ -0,0 +1,124 @@
+from asgiref.sync import async_to_sync
+from django.core.management import BaseCommand
+from django.core.paginator import Paginator
+from django.db import transaction
+
+from cl.lib.command_utils import logger
+from cl.lib.microservice_utils import microservice
+from cl.search.models import (
+    SOURCES,
+    Court,
+    Docket,
+    Opinion,
+    OpinionCluster,
+    RECAPDocument,
+)
+
+
+def import_opinions_from_recap(court=None, total_count=0):
+    """Import recap documents into opinion db
+
+    :param court: Court ID if any
+    :param total_count: The number of new opinions to add
+    :return: None
+    """
+    count = 0
+    if not court:
+        courts = Court.objects.filter(jurisdiction=Court.FEDERAL_DISTRICT)
+    else:
+        courts = Court.objects.filter(id=court)
+    for court in courts:
+        logger.info(f"Importing RECAP documents for {court}")
+        cluster = (
+            OpinionCluster.objects.filter(docket__court=court)
+            .order_by("-date_filed")
+            .first()
+        )
+        if not cluster:
+            print("No clusters for this court")
+            logger.info(f"Court {court} has no opinion clusters")
+
+            continue
```

Comment:
Misplaced newline here, but can you explain why we skip courts with no cluster objects? Seems like we'd particularly want to add content to such courts, no?

Reply:
We have one court, the District Court for District of Orleans that has no opinions.  I think I was trying to avoid issues from historical courts.  

https://en.wikipedia.org/wiki/United_States_District_Court_for_the_District_of_Orleans

Reply:
In that case, it's better to be explicit and exclude it from your query. In a year that code will still be clear of its intentions.

Reply:
sounds good